.github/workflows/ai-automation.yml

```
@@ -0,0 +1,138 @@
+name: '🤖 AI Automation'
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)